app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args: Callable) -> Callable:
```

Comment:
in this case better to use other type annotation to function return
u return not callable, u return result from func()